docs/components/nav_rail.md

```diff
@@ -0,0 +1,107 @@
+<!--docs:
+title: "Navigation Rail"
+layout: detail
+section: components
+excerpt: "Navigation rails provide ergonomic movement between primary destinations in apps running on a large screen device"
+iconId: navigation_rail
+path: /catalog/navigation-rail/
+-->
+
+# Navigation Rail
+
+[Navigation rail](https://material.io/components/navigation-rail/#) allows
+movement between primary destinations in an app.
+
+![The navigation rail container is 72 dp wide by default.](assets/nav_rail/navigation-rail-default.png)
+
+**Contents**
+
+*   [Using navigation rail](#using-navigation-rail)
+*   [Navigation rail example](#navigation-rail-example)
+*   [Theming](#theming-a-navigation-rail)
+
+## Using navigation rail
+
+Before you can a navigation rail, you need to import the Material Components package for Flutter:
+
+```dart
+package:flutter/material.dart.
+```
+
+You need to be using a [`MaterialApp`](https://api.flutter.dev/flutter/material/MaterialApp-class.html).
+
+For more information on getting started with the Material for Flutter, go to the Flutter [Material library](https://api.flutter.dev/flutter/material/material-library.html) page.
+
+### Making navigation rail accessible
+
+Flutter's drawer component APIs support labeling for accessibility via the `semanticLabel` property. The content within `child` can have additional accessibility labeling.
```

Comment:
I think its a little different here. the labels are in NavigationRailDestinations (which just take in widgets), but the rail will attempt to include semantics automatically

Reply:
What do you think would be a good paragraph to replace this with?

Reply:
Flutter's navigation rail API supports labeling for accessibility via `NavigationRailDestination.label`. This `label` is often a `Text`, which automatically includes semantics. In the rare case where the `label` is not a `Text`, it can be wrapped in `Semantics` with a custom label string.

Reply:
Thanks!